Biblio.bib

```diff
@@ -113,6 +118,7 @@ @InProceedings{dwork92pricingvia
 }
 
 @Article{nakamoto2008bitcoin,
+	url = {{http://nakamotoinstitute.org/bitcoin/}},
```

Comment:
Let's follow [the orignal author](https://www.mail-archive.com/cryptography@metzdowd.com/msg09959.html) and use
`http://www.bitcoin.org/bitcoin.pdf`

Reply:
OK, I hadn't seen that one.

Reply:
Let's use https://www.mail-archive.com/cryptography@metzdowd.com/msg09959.html?